packages/psss-server/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@tupaia/psss-server",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Server for building report data",
+  "homepage": "https://github.com/beyondessential/tupaia",
+  "bugs": {
+    "url": "https://github.com/beyondessential/tupaia/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/beyondessential/tupaia"
+  },
+  "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "babel src --out-dir dist --source-maps --ignore 'src/__tests__/**' --extensions '.ts' --config-file '../../.babelrc-ts.js'",
+    "lint": "tsc; eslint . --ext .ts",
+    "lint:all": "yarn run lint \"src/**/*.{ts,jsx}\"",
+    "lint:fix": "yarn lint --fix",
+    "prestart": "npm run -s build",
+    "start": "node dist",
+    "start-dev": "../../scripts/bash/backendStartDev.sh 9995 -ts",
```

```suggestion
    "start-dev": "../../scripts/bash/backendStartDev.sh 9991 -ts",
```
May want to change the debugger port, incase you want to run `report-server` and `psss-server` at the same time?

---

Good call